pom.xml

```diff
@@ -854,6 +854,15 @@
   <build>
       <pluginManagement>
           <plugins>
+              <plugin>
+                  <groupId>org.kohsuke</groupId>
+                  <artifactId>access-modifier-checker</artifactId>
```

Comment:
@olamy probably ok. @jglick does it introduce/mask problems it shouldn't?

Reply:
Yes.

Reply:
@jglick ok, so I take it as not a good idea to introduce. @olamy what its solving? What kind of error we are trying to fix with this?

Reply:
BlueUrlTokenizer (module blueocean-commons) is marked with @Restricted(NoExternalUse.class).
This is used by many classes outside this module (PipelineActivityStatePreloader from blueocean-pipeline-api-impl etc..)
I guess it's an internal use :-)
So here few solutions:
1. skip access-modifier-checker
2. rewrite/remove BlueUrlTokenizer
3. fix access-modifier-checker plugin to accept usage with @Restricted(NoExternalUse.class) from modules within the same project (reactors)

I would say 3 (I need to have a look how long will be the fix)
Let me know what you prefer?

Reply:
I have implemented/pushed solution 4.